keyboards/momokai/aurora/config.h

```diff
@@ -20,7 +20,7 @@
 //#define NO_ACTION_ONESHOT
 
 
-#define RGB_MATRIX_LED_COUNT 10
+#define RGB_MATRIX_LED_COUNT 6
```

Comment:
It looks like there are 4 leds around the encoder.  Is this not the case?

Reply:
winry decided to remove those 4 leds, since there's no transparency on the encoder knob and the enclosure around the knob isn't designed for it either